File: Modulo14/Funcionario-Java/Enfermeiro.java

public class Enfermeiro extends Funcionario{

    private int CRE; // Conselho Regional de Enfermagem

    public Enfermeiro(int CPF, String nome, int CRE) {
        super(CPF, nome);
        this.CRE = CRE;
    }

    public String getProfissao() {
        return "Enfermeiro";
    }

    public void imprime() {
        super.imprime();
        System.out.println("CRE: " + CRE);
    } 
}

File: Modulo12/Funcionario-CSharp/Enfermeiro.cs
using System;

namespace poo {
    public class Enfermeiro : Funcionario {

        private int CRE; // Conselho Regional de Enfermagem

        public Enfermeiro(string CPF, string nome, int CRE) : base(CPF, nome) {
            this.CRE = CRE;
        }

        public override string getProfissao() {
            return "Enfermeiro";
        }

        public int getCRE() {
            return CRE;
        }

        public override void imprime() {
            base.imprime();
            Console.WriteLine("CRE: {0}", this.CRE);
        }
    }
}